app/src/main/java/com/devshish/internship/domain/model/Song.kt

```diff
@@ -1,8 +1,11 @@
 package com.devshish.internship.domain.model
 
+import android.net.Uri
+
 data class Song(
+    val uri: Uri?,
     val title: String,
     val artist: String,
-    val duration: Int,
+    val duration: Long,
```

Comment:
I think that Int type will be enough - Long contains too much for song duration(even in ms)